src/common.ts

```diff
@@ -24,7 +24,15 @@ export const endpointMap = {
   getUserEvents: `${BASE_URL}/v3/events`,
   deleteUser: `${BASE_URL}/v1/user`,
   listDeployments: `${BASE_URL}/v6/deployments`,
+  listDeploymentFiles: (id: string) => `${BASE_URL}/v6/deployments/${id}/files`,
```

Comment:
We're using qualified `ids` now so instead of `id` we'll use `deploymentId` here